pack.go

```diff
@@ -71,7 +71,12 @@ const (
 	// PackManifestVersion1_1_RC4 represents the OCI Image Manifest defined
 	// in image-spec v1.1.0-rc4.
 	// Reference: https://github.com/opencontainers/image-spec/blob/v1.1.0-rc4/manifest.md
-	PackManifestVersion1_1_RC4 PackManifestVersion = 2
+	PackManifestVersion1_1_RC4 PackManifestVersion = PackManifestVersion1_1
```

Comment:
This constant should be marked as deprecated.

Reply:
deprecated this constant.